### PR TITLE
site: add Posthog

### LINKED ISSUE #25: Add analytics tracking

### PR DESCRIPTION
**Pull request by luwes**
fix #25 

https://player-style-git-posthog-mux.vercel.app/

### CODE REVIEW

File: site/app/_components/AnalyticsProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import posthog from 'posthog-js';
+import { PostHogProvider } from 'posthog-js/react';
+
+if (typeof window !== 'undefined') {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+    api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST,
+    person_profiles: 'identified_only',
```
Comment (mmcc) [inline site/app/_components/AnalyticsProvider.tsx:9]: I think we probably want this to be "always" here since we'll never be identifying anyone on this site.

Comment (luwes) [reply site/app/_components/AnalyticsProvider.tsx]: updated!

File: site/app/(home)/page.tsx [hunk 2]
```diff
@@ -19,9 +20,9 @@ export default async function Home({ searchParams }: HomeProps) {
     <>
       <Hero title="Create your Player">
         Look on our player theme treasures; compatible with most web video and audio players.{' '}
-        <a href="/about" className="underline">
+        <Link href="/about" className="underline">
```
Comment (heff) [inline site/app/(home)/page.tsx:23]: Was this change necessary for post hog for some reason, or just a side fix?

Comment (luwes) [reply site/app/(home)/page.tsx:23]: it wasn't required. a bit related in that Posthog adds the tracking to Next.js to accommodate pjax style navigation and that made me double check all the nav links were proper
